src/Security/ApiKeyAuthenticator.php

```diff
@@ -14,7 +14,10 @@
 
 class ApiKeyAuthenticator implements SimplePreAuthenticatorInterface
 {
-    public function createToken(Request $request, $providerKey)
+    /**
+     * @param string $providerKey
+     */
+    public function createToken(Request $request, $providerKey): PreAuthenticatedToken
```

Comment:
The ApiKeyAuthenticator is still extending [SimpleAuthenticatorInterface](https://github.com/symfony/symfony/blob/4.4/src/Symfony/Component/Security/Core/Authentication/SimpleAuthenticatorInterface.php)

Deprecated and we should use guard, but for now we can not say: string $providerKey because it conflicts with the interface.